x-pack/plugins/security_solution/public/resolver/types.ts

```diff
@@ -227,6 +227,22 @@ export interface NodeEventsInCategoryState {
    * The cursor, if any, that can be used to retrieve more events.
    */
   cursor: null | string;
+
+  /**
+   * The cursor, if any, that was last used to fetch additional related events.
+   */
+
+  lastCursorRequested?: null | string;
```

Comment:
This field can be null or undefined. If the feature relies on that, can you explain in the doc comment how? Otherwise can you explain when it would be null vs undefined and why it doesn't matter?